backend/src/main/java/de/nordakademie/iaa/noodle/dao/SurveyRepository.java

```diff
@@ -53,10 +53,10 @@
                (:participated = false AND
                    (NOT EXISTS (SELECT p FROM Participation p WHERE p.survey=survey AND p.participant.id = :userID))))
            AND
-           (:completed IS NULL OR
-               (:completed=true  AND NOT
+           (:closed IS NULL OR
```

Comment:
Not really your fault here, but since you're touching it anyway, just a food for thought: Do we want to keep this explicit as `:completed = true` / `false` or should we simplify to `:completed` / `NOT :completed`

Reply:
Good idea. I'll change that.

Reply:
That does not seem to work. When I change `:completed=true` to `:completed`, the server throws a `BeanCreationException`. The same happens for the negated case.

Reply:
Alright, we'll keep it this way.